ci/taos/plugins-base/pr-audit-build-android.sh

```diff
@@ -140,10 +141,13 @@ function pr-audit-build-android-run-queue(){
         # www-data    ALL=(ALL) NOPASSWD:ALL
         echo -e "[DEBUG] current folder is $(pwd)."
 
+        # NNStreamer root directory for build.
+        export NNSTREAMER_ROOT=$(pwd)
```

Comment:
@kparichay , Why do you want to use "**export**" in this script file? What is the purpose of that?

Reply:
"export" was being used earlier on line 197
I didn't want to change that

Reply:
Synched. Let's keep the "export" style.